MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -334,8 +304,8 @@ class HomeViewController: UIViewController, FloatingPanelControllerDelegate, Tra
         addChild(fpc)
 
         //Move send button to in front of panel
-        bottomFadeView.superview?.bringSubviewToFront(bottomFadeView)
-        sendButton.superview?.bringSubviewToFront(sendButton)
+        self.bottomFadeView.superview?.bringSubviewToFront(self.bottomFadeView)
```

Comment:
why adding self here? It is the same thing than before. I think as a good rule of thumb we should use self only inside closures. You can leave it like this. i'm just curious why you changed this adding self.

Reply:
Must be a copy/paste error, will fix